### ISSUE TITLE
Partial match of various elements in `prior` object

### ISSUE THREAD

**Issue by jamesmbaazam** [epinowcast/epinowcast#472]
When I run `epinowcast()`, I get the following partial matching warning, suggesting there might be an issue with using the wrong variable in the `priors` object. I'm not sure if this is a problem or not and I'm happy for you to close the issue if it's a false alarm.  

``` r
# Turn on partial matching options
options(
  warnPartialMatchAttr = TRUE,
  warnPartialMatchDollar = TRUE,
  warnPartialMatchArgs = TRUE
)
remotes::install_github("epinowcast/epinowcast", dependencies = TRUE)
#> Using GitHub PAT from the git credential store.
#> Skipping install of 'epinowcast' from a github remote, the SHA1 (886b45cb) has not changed since last install.
#>   Use `force = TRUE` to force installation
```

``` r
library(epinowcast)
#> ℹ Using `/Users/lshja16/Library/Caches/org.R-project.R/R/epinowcast` for the
#>   epinowcast model cache location.
#> This message is displayed once per session.
```

``` r

pobs <- enw_example("preprocessed")
latest_obs <- enw_example("observations")

path_nowcast <- epinowcast(pobs,
  expectation = enw_expectation(~1, data = pobs),
  fit = enw_fit_opts(enw_pathfinder, pp = TRUE, num_threads = 4, num_paths = 16),
  obs = enw_obs(family = "poisson", data = pobs)
)
#> ℹ Using model /Users/lshja16/Library/R/arm64/4.4/library/epinowcast/stan/epinowcast.stan.
#> ℹ Include is /Users/lshja16/Library/R/arm64/4.4/library/epinowcast/stan.
#> ℹ Using `/Users/lshja16/Library/Caches/org.R-project.R/R/epinowcast` for the
#>   epinowcast model cache location.
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$expr_r_int: partial match of 'expr_r_int' to 'expr_r_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_mean_int: partial match of 'refp_mean_int' to
#> 'refp_mean_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Warning in priors$refp_sd_int: partial match of 'refp_sd_int' to
#> 'refp_sd_int_p'
#> Path [1] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [1] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [1] :Best Iter: [31] ELBO (-1998.710095) evaluations: (826) 
#> Path [2] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [2] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [2] :Best Iter: [27] ELBO (-1998.675299) evaluations: (826) 
#> Path [3] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [3] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [3] :Best Iter: [30] ELBO (-1998.585015) evaluations: (826) 
#> Path [4] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [4] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [4] :Best Iter: [30] ELBO (-1998.644494) evaluations: (826) 
#> Path [5] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [5] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [5] :Best Iter: [31] ELBO (-1998.590865) evaluations: (826) 
#> Path [6] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [6] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [6] :Best Iter: [32] ELBO (-1998.649508) evaluations: (826) 
#> Path [7] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [7] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [7] :Best Iter: [27] ELBO (-1998.657261) evaluations: (826) 
#> Path [8] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [8] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [8] :Best Iter: [32] ELBO (-1998.597835) evaluations: (826) 
#> Path [9] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [9] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [9] :Best Iter: [31] ELBO (-1998.655538) evaluations: (826) 
#> Path [10] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [10] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [10] :Best Iter: [32] ELBO (-1998.545939) evaluations: (826) 
#> Path [11] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [11] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [11] :Best Iter: [30] ELBO (-1998.680496) evaluations: (826) 
#> Path [12] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [12] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [12] :Best Iter: [30] ELBO (-1998.619750) evaluations: (826) 
#> Path [13] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [13] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [13] :Best Iter: [32] ELBO (-1998.675283) evaluations: (826) 
#> Path [14] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [14] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [14] :Best Iter: [33] ELBO (-1998.650792) evaluations: (826) 
#> Path [15] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [15] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [15] :Best Iter: [27] ELBO (-1998.609702) evaluations: (826) 
#> Path [16] :Initial log joint density = -13292.856175 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Error evaluating model log probability: Non-finite gradient. 
#> Path [16] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes  
#>              33      -1.983e+03      3.577e-04   1.127e+00    1.000e+00  1.000e+00       826 -1.999e+03 -1.999e+03                   
#> Path [16] :Best Iter: [32] ELBO (-1998.653824) evaluations: (826) 
#> Total log probability function evaluations:28816 
#> Finished in  3.7 seconds.
```

<sup>Created on 2024-06-26 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by Bisaloo** [comment]
As you guessed it, this seems to be fixed by #343. Could you double check if you agree and close this issue if you do please?

**Comment by jamesmbaazam** [comment]
Yes, it's fixed now. Thanks. I'll now close this issue.
